shared/actions/login/index.js

```diff
@@ -359,41 +373,58 @@ function makeKex2IncomingMap (dispatch, getState, provisionMethod, userPassTitle
         response.result(username)
       }
     },
-    'keybase.1.secretUi.getPaperKeyPassphrase': (param, response) => {
```

Comment:
these were all deprecated and replaced by a single call. i'm just making paper key work first, then i'll do the rest next